src/Illuminate/Console/ContainerCommandLoader.php

```diff
@@ -17,23 +17,33 @@ class ContainerCommandLoader implements CommandLoaderInterface
     protected $container;
 
     /**
-     * A map of command names to classes.
+     * A list of class names.
      *
      * @var array
      */
-    protected $commandMap;
+    protected $classes;
 
     /**
      * Create a new command loader instance.
      *
      * @param  \Psr\Container\ContainerInterface  $container
-     * @param  array  $commandMap
+     * @param  array  $classes
```

Comment:
There is no second parameter here?

Reply:
The second parameter was removed - I've now deleted that docblock comment.